### PR TITLE
Change AgentBinaries URL

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes a part of #1974 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by testing the endpoint manually on Windows.
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

### CODE REVIEW

File: monkey/monkey_island/cc/resources/agent_binaries.py [hunk 1]
```diff
@@ -20,9 +20,7 @@ class UnsupportedOSError(Exception):
 
 
 class AgentBinaries(AbstractResource):
-    # API Spec: Rename to /api/agent-binaries, because information about agent runs
-    # and binary files are different resources
-    urls = ["/api/agent/download/<string:host_os>"]
+    urls = ["/api/agent-binaries/<string:host_os>"]
```
Comment (mssalvatore) [inline monkey/monkey_island/cc/resources/agent_binaries.py:23]:
```suggestion
    urls = ["/api/agent-binaries/<string:os>"]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,8 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - All "/api/monkey_control" endpoints to "/api/monkey-control". #1888
 - All "/api/monkey" endpoints to "/api/agent". #1888
 - Update MongoDB version to 4.4.x. #1924
+- Endpoint to get agent binaries from "/api/agent/download/<string:os>" to
```
Comment (VakarisZ) [inline CHANGELOG.md:37]: I don't think this is relevant to the users

**Review by VakarisZ — APPROVED** [review]
One small comment

File: CHANGELOG.md (hunk 2 above)
Comment (shreyamalviya) [reply CHANGELOG.md:37]: Besides being an API endpoint which seems relevant enough to me, it is encountered by the user if they try to run the monkey manually using the command given in the UI. I feel like it is.

Comment (mssalvatore) [reply CHANGELOG.md:37]: Changes for any user-facing interface belong in the changelog.

**Pull request merged**